cluster/gce/gci/configure-helper.sh

```diff
@@ -279,6 +279,11 @@ function create-master-auth {
   local -r auth_dir="/etc/srv/kubernetes"
   local -r basic_auth_csv="${auth_dir}/basic_auth.csv"
   if [[ -n "${KUBE_PASSWORD:-}" && -n "${KUBE_USER:-}" ]]; then
+    if [[ -e "${basic_auth_csv}" && "${METADATA_CLOBBERS_CONFIG:-false}" == "true" ]]; then
+      sed -i "/,${KUBE_USER},admin,system:masters$/d" "${basic_auth_csv}"
```

Comment:
@roberthbailey Yes the primary reason we cannot use replace_prefixed_line is because we do not know the "old" password. In addition that method only allows you to replace the sufffix given a prefix. In this case we want to replace the prefix given a full match on the suffic and a partial match on the prefix.